lib/core/services/authentication.service.ts

```diff
@@ -229,6 +229,19 @@ export class AuthenticationService {
         return this.alfrescoApi.getInstance().bpmAuth.username;
     }
 
+    /**
+     * Get the BPM username
+     *
+     * @returns {User} the User Info
+     *
+     * @memberof AuthenticationService
+     */
+    getCurrentUserInfo(): Observable<any> {
```

Comment:
there is already a service that does this thing. is the bpm user service in the userinfo. Please don't duplicate call and this is not the right place for this call is not concern of the authentication.